docs/UserGuide.md

```diff
@@ -213,12 +213,14 @@ The different types of `CATEGORY` are listed below:
   
 Any `CATEGORY` that does not falls in the list could be put under `Miscellaneous`.
 
-All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#342-list-recipes-listrecipe)  
+> All `RECIPE_NAME` has to be unique. You can check the list of existing recipes by using [`listrecipe all`](#332-list-ingredient-listingredient)
```

Comment:
@alwayshuizhen `listrecipe all` linking to `listrecipe header` or `listingredient header`?